build/repo.props

```diff
@@ -8,7 +8,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <DotNetCoreRuntime Include="$(MicrosoftNETCoreApp20PackageVersion)" />
     <DotNetCoreRuntime Include="$(MicrosoftNETCoreApp21PackageVersion)" />
   </ItemGroup>
+
+  <ItemGroup>
+   <AspNetCoreRuntime Include="$(MicrosoftAspNetCoreAllPackageVersion)" />
```

Comment:
You probably need to also update dependencies.props. MicrosoftAspNetCoreAllPackageVersion is set to version before we started producing the shared framework.

Reply:
Yeah, just realized that. I'm adding that.